internal/impl/packages.go

```diff
@@ -33,9 +33,17 @@ func (d *Devbox) Add(ctx context.Context, pkgs ...string) error {
 	ctx, task := trace.NewTask(ctx, "devboxAdd")
 	defer task.End()
 
-	pkgs = lo.Uniq(pkgs)
-
-	original := d.cfg.Packages
+	versionedPackages := []string{}
+	// Add to Packages of the config only if it's not already there. We do this
+	// before addin @latest to ensure we don't accidentally add a package that
+	// is already in the config.
+	for _, pkg := range nix.InputsFromStrings(lo.Uniq(pkgs), d.lockfile) {
```

Comment:
can we call `pkg` as `input` here? In the context of this function since the `pkgs ...string` param is there, then I'll always assume that `pkg` is of type `string`.